main/urls.py

```diff
@@ -105,6 +110,7 @@
     url(r'^sendperform', FormSent.as_view()),
     url(r'^editperform', FormEdit.as_view()),
     url(r'^sendperdraft', DraftSent.as_view()),
+    url(r'^sendproject', ProjectSent.as_view()),
```

Comment:
@szabozoltan69 for naming this URL and view, can we follow a similar naming as the `CreateFieldReport`

So something like:

     url(r'^api/v2/create_project/', api_views.CreateProject.as_view())

It would be nice to have this under the `/api/` URLs and `CreateProject` for the name of the view is a bit clear what it's doing.

We can leave the other URLs as is, of course.